lfs/ssh.go

```diff
@@ -37,12 +38,22 @@ func sshAuthenticate(endpoint Endpoint, operation, oid string) (sshAuthResponse,
 
 	cmd := exec.Command(exe, args...)
 
-	out, err := cmd.CombinedOutput()
+	// Save stdout and stderr in separate buffers
+	var outbuf, errbuf bytes.Buffer
+	cmd.Stdout = &outbuf
+	cmd.Stderr = &errbuf
```

Comment:
I dont know. This fragment was found on StackOverflow: http://stackoverflow.com/questions/31095699/return-cmd-stdout-and-stderr-as-string-instead-of-printing-to-console-in-golang/31095907
I have no golang experience.